nibabel/tests/test_diff.py

```diff
@@ -32,17 +33,58 @@
 from .test_parrec_data import BALLS, AFF_OFF
 from .test_helpers import assert_data_similar
 
+from hypothesis import given
+import hypothesis.strategies as st
 
 
 DATA_PATH = abspath(pjoin(dirname(__file__), 'data'))
 
 from nibabel.cmdline.diff import diff_dicts
 
 
-def test_diff_dicts():
-    assert_equal(diff_dicts('key', None, None), None)
-    assert_equal(diff_dicts('dtype', int, float), {'dtype': (int, float)})
-    assert_equal(diff_dicts(1, 2, 2), None)
-    assert_equal(diff_dicts('hdr', 1, None), {'hdr': (1, None)})
-    assert_equal(diff_dicts('array', np.array([[-3.,  0.,  0.,  3.]]), np.array([[-2.,  0.,  0.,  3.]])), {'array': (array([[-3.,  0.,  0.,  3.]]), array([[-2.,  0.,  0.,  3.]]))})
-    # TODO: mixed cases
\ No newline at end of file
+@given(st.data())
```

Comment:
how many cases this `@given` runs through I wonder? i.e. among that whooping 1.5 sec, which tests consume most and either there is some centralized way to control how many strategies hypothesis tests "by default"?

Reply:
The documentation states that the default number of cases is 100 but I'm not really buying it. Either way, it can be adjusted. See "max_examples" here for more information: https://hypothesis.readthedocs.io/en/latest/settings.html